stacks-signer/src/cli.rs

```diff
@@ -139,6 +143,9 @@ pub struct RunDkgArgs {
     /// Path to config file
     #[arg(long, value_name = "FILE")]
     pub config: PathBuf,
+    /// The reward cycle the signer is registered for and wants to peform DKG for
+    #[arg(long, short)]
+    pub reward_cycle: u64,
```

Comment:
Same question for this one.

Reply:
So this is actually to enable the signer_set calculation to know which signer we are sending this to (a single signer runs two simultaneous signers and it needs to know which signer should receive the command)